### ISSUE TITLE
2025-05-16, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#171]
# Today's Arxiv Papers

(1) [End-to-End Vision Tokenizer Tuning](http://arxiv.org/abs/2505.10562v1), 2025-05-15

(2) [Neural Thermodynamic Laws for Large Language Model Training](http://arxiv.org/abs/2505.10559v1), 2025-05-15

(3) [Does Feasibility Matter? Understanding the Impact of Feasibility on Synthetic Training Data](http://arxiv.org/abs/2505.10551v1), 2025-05-15

(4) [Towards a Deeper Understanding of Reasoning Capabilities in Large Language Models](http://arxiv.org/abs/2505.10543v1), 2025-05-15

(5) [Exploring Implicit Visual Misunderstandings in Multimodal Large Language Models through Attention Analysis](http://arxiv.org/abs/2505.10541v1), 2025-05-15

(6) [S3C2 Summit 2024-09: Industry Secure Software Supply Chain Summit](http://arxiv.org/abs/2505.10538v1), 2025-05-15

(7) [RouteNator: A Router-Based Multi-Modal Architecture for Generating Synthetic Training Data for Function Calling LLMs](http://arxiv.org/abs/2505.10495v1), 2025-05-15

(8) [Can You Really Trust Code Copilots? Evaluating Large Language Models from a Code Security Perspective](http://arxiv.org/abs/2505.10494v1), 2025-05-15

(9) [CL-RAG: Bridging the Gap in Retrieval-Augmented Generation with Curriculum Learning](http://arxiv.org/abs/2505.10493v1), 2025-05-15

(10) [Campus AI vs Commercial AI: A Late-Breaking Study on How LLM As-A-Service Customizations Shape Trust and Usage Patterns](http://arxiv.org/abs/2505.10490v1), 2025-05-15

(11) [Large Language Models for Cancer Communication: Evaluating Linguistic Quality, Safety, and Accessibility in Generative AI](http://arxiv.org/abs/2505.10472v1), 2025-05-15

(12) [AI Agents vs. Agentic AI: A Conceptual Taxonomy, Applications and Challenge](http://arxiv.org/abs/2505.10468v1), 2025-05-15

(13) [Superposition Yields Robust Neural Scaling](http://arxiv.org/abs/2505.10465v1), 2025-05-15

(14) [Are Large Language Models Robust in Understanding Code Against Semantics-Preserving Mutations?](http://arxiv.org/abs/2505.10443v1), 2025-05-15

(15) [Learning to Think: Information-Theoretic Reinforcement Fine-Tuning for LLMs](http://arxiv.org/abs/2505.10425v1), 2025-05-15

(16) [Are LLM-generated plain language summaries truly understandable? A large-scale crowdsourced evaluation](http://arxiv.org/abs/2505.10409v1), 2025-05-15

(17) [Rethinking Repetition Problems of LLMs in Code Generation](http://arxiv.org/abs/2505.10402v1), 2025-05-15

(18) [Multi-domain Multilingual Sentiment Analysis in Industry: Predicting Aspect-based Opinion Quadruples](http://arxiv.org/abs/2505.10389v1), 2025-05-15

(19) [Are Sparse Autoencoders Useful for Java Function Bug Detection?](http://arxiv.org/abs/2505.10375v1), 2025-05-15

(20) [FactsR: A Safer Method for Producing High Quality Healthcare Documentation](http://arxiv.org/abs/2505.10360v1), 2025-05-15

(21) [LDIR: Low-Dimensional Dense and Interpretable Text Embeddings with Relative Representations](http://arxiv.org/abs/2505.10354v1), 2025-05-15

(22) [AutoPentest: Enhancing Vulnerability Management With Autonomous LLM Agents](http://arxiv.org/abs/2505.10321v1), 2025-05-15

(23) [J1: Incentivizing Thinking in LLM-as-a-Judge via Reinforcement Learning](http://arxiv.org/abs/2505.10320v1), 2025-05-15

(24) [Empirically evaluating commonsense intelligence in large language models with large-scale human judgments](http://arxiv.org/abs/2505.10309v1), 2025-05-15

(25) [AI LEGO: Scaffolding Cross-Functional Collaboration in Industrial Responsible AI Practices during Early Design Stages](http://arxiv.org/abs/2505.10300v1), 2025-05-15

(26) [From Questions to Clinical Recommendations: Large Language Models Driving Evidence-Based Clinical Decision Making](http://arxiv.org/abs/2505.10282v1), 2025-05-15

(27) [MASS: Multi-Agent Simulation Scaling for Portfolio Construction](http://arxiv.org/abs/2505.10278v1), 2025-05-15

(28) [The Evolving Landscape of Generative Large Language Models and Traditional Natural Language Processing in Medicine](http://arxiv.org/abs/2505.10261v1), 2025-05-15

(29) [Comparing LLM Text Annotation Skills: A Study on Human Rights Violations in Social Media Data](http://arxiv.org/abs/2505.10260v1), 2025-05-15

(30) [SpecOffload: Unlocking Latent GPU Capacity for LLM Inference on Resource-Constrained Devices](http://arxiv.org/abs/2505.10259v1), 2025-05-15

(31) [RAIDEN-R1: Improving Role-awareness of LLMs via GRPO with Verifiable Reward](http://arxiv.org/abs/2505.10218v1), 2025-05-15

(32) [Informed Forecasting: Leveraging Auxiliary Knowledge to Boost LLM Performance on Time Series Forecasting](http://arxiv.org/abs/2505.10213v1), 2025-05-15

(33) [Do LLMs Memorize Recommendation Datasets? A Preliminary Study on MovieLens-1M](http://dx.doi.org/10.1145/3726302.3730178), 2025-05-15

(34) [VQ-Logits: Compressing the Output Bottleneck of Large Language Models via Vector Quantized Logits](http://arxiv.org/abs/2505.10202v1), 2025-05-15

(35) [The CoT Encyclopedia: Analyzing, Predicting, and Controlling how a Reasoning Model will Think](http://arxiv.org/abs/2505.10185v1), 2025-05-15

(36) [Mining Hidden Thoughts from Texts: Evaluating Continual Pretraining with Synthetic Data for LLM Reasoning](http://arxiv.org/abs/2505.10182v1), 2025-05-15

(37) [GE-Chat: A Graph Enhanced RAG Framework for Evidential Response Generation of LLMs](http://arxiv.org/abs/2505.10143v1), 2025-05-15

(38) [Learning Virtual Machine Scheduling in Cloud Computing through Language Agents](http://arxiv.org/abs/2505.10117v1), 2025-05-15

(39) [What Does Neuro Mean to Cardio? Investigating the Role of Clinical Specialty Data in Medical LLMs](http://arxiv.org/abs/2505.10113v1), 2025-05-15

(40) [From Text to Network: Constructing a Knowledge Graph of Taiwan-Based China Studies Using Generative AI](http://arxiv.org/abs/2505.10093v1), 2025-05-15

(41) [XRAG: Cross-lingual Retrieval-Augmented Generation](http://arxiv.org/abs/2505.10089v1), 2025-05-15

(42) [ChronoSteer: Bridging Large Language Model and Time Series Foundation Model via Synthetic Data](http://arxiv.org/abs/2505.10083v1), 2025-05-15

(43) [Leveraging Graph Retrieval-Augmented Generation to Support Learners' Understanding of Knowledge Concepts in MOOCs](http://arxiv.org/abs/2505.10074v1), 2025-05-15

(44) [Dark LLMs: The Growing Threat of Unaligned AI Models](http://arxiv.org/abs/2505.10066v1), 2025-05-15

(45) [CAFE: Retrieval Head-based Coarse-to-Fine Information Seeking to Enhance Multi-Document QA Capability](http://arxiv.org/abs/2505.10063v1), 2025-05-15

(46) [Exploring the Deep Fusion of Large Language Models and Diffusion Transformers for Text-to-Image Synthesis](http://arxiv.org/abs/2505.10046v1), 2025-05-15

(47) [DIF: A Framework for Benchmarking and Verifying Implicit Bias in LLMs](http://arxiv.org/abs/2505.10013v1), 2025-05-15

(48) [ImagineBench: Evaluating Reinforcement Learning with Large Language Model Rollouts](http://arxiv.org/abs/2505.10010v1), 2025-05-15

(49) [SVA-ICL: Improving LLM-based Software Vulnerability Assessment via In-Context Learning and Information Fusion](http://arxiv.org/abs/2505.10008v1), 2025-05-15

(50) [AI2MMUM: AI-AI Oriented Multi-Modal Universal Model Leveraging Telecom Domain Large Model](http://arxiv.org/abs/2505.10003v1), 2025-05-15

(51) [ServeGen: Workload Characterization and Generation of Large Language Model Serving in Production](http://arxiv.org/abs/2505.09999v1), 2025-05-15

(52) [Analysing Safety Risks in LLMs Fine-Tuned with Pseudo-Malicious Cyber Security Data](http://arxiv.org/abs/2505.09974v1), 2025-05-15

(53) [Pre-Act: Multi-Step Planning and Reasoning Improves Acting in LLM Agents](http://arxiv.org/abs/2505.09970v1), 2025-05-15

(54) [Advanced Crash Causation Analysis for Freeway Safety: A Large Language Model Approach to Identifying Key Contributing Factors](http://arxiv.org/abs/2505.09949v1), 2025-05-15

(55) [Personalizing Large Language Models using Retrieval Augmented Generation and Knowledge Graph](http://dx.doi.org/10.1145/3701716.3715473), 2025-05-15

(56) [Design and Evaluation of Generative Agent-based Platform for Human-Assistant Interaction Research: A Tale of 10 User Studies](http://arxiv.org/abs/2505.09938v1), 2025-05-15

(57) [CartoAgent: a multimodal large language model-powered multi-agent cartographic framework for map style transfer and evaluation](http://arxiv.org/abs/2505.09936v1), 2025-05-15

(58) [Rethinking Prompt Optimizers: From Prompt Merits to Optimization](http://arxiv.org/abs/2505.09930v1), 2025-05-15

(59) [Reinforced Interactive Continual Learning via Real-time Noisy Human Feedback](http://arxiv.org/abs/2505.09925v1), 2025-05-15

(60) [From Trade-off to Synergy: A Versatile Symbiotic Watermarking Framework for Large Language Models](http://arxiv.org/abs/2505.09924v1), 2025-05-15

(61) [PIG: Privacy Jailbreak Attack on LLMs via Gradient-based Iterative In-Context Optimization](http://arxiv.org/abs/2505.09921v1), 2025-05-15

(62) [UICopilot: Automating UI Synthesis via Hierarchical Code Generation from Webpage Designs](http://dx.doi.org/10.1145/3696410.3714891), 2025-05-15

(63) [Crossing Borders Without Crossing Boundaries: How Sociolinguistic Awareness Can Optimize User Engagement with Localized Spanish AI Models Across Hispanophone Countries](http://arxiv.org/abs/2505.09902v1), 2025-05-15

(64) [Comparing Exploration-Exploitation Strategies of LLMs and Humans: Insights from Standard Multi-armed Bandit Tasks](http://arxiv.org/abs/2505.09901v1), 2025-05-15

(65) [Characterizing Unintended Consequences in Human-GUI Agent Collaboration for Web Browsing](http://arxiv.org/abs/2505.09875v1), 2025-05-15

(66) [Do Large Language Models Know Conflict? Investigating Parametric vs. Non-Parametric Knowledge of LLMs for Conflict Forecasting](http://arxiv.org/abs/2505.09852v1), 2025-05-14

(67) [Evaluating Large Language Models for the Generation of Unit Tests with Equivalence Partitions and Boundary Values](http://arxiv.org/abs/2505.09830v1), 2025-05-14

(68) [KRISTEVA: Close Reading as a Novel Task for Benchmarking Interpretive Reasoning](http://arxiv.org/abs/2505.09825v1), 2025-05-14

(69) [Adversarial Attack on Large Language Models using Exponentiated Gradient Descent](http://arxiv.org/abs/2505.09820v1), 2025-05-14

(70) [Lossless Compression for LLM Tensor Incremental Snapshots](http://arxiv.org/abs/2505.09810v1), 2025-05-14

(71) [Exploring the generalization of LLM truth directions on conversational formats](http://arxiv.org/abs/2505.09807v1), 2025-05-14

(72) [Learn, Explore and Reflect by Chatting: Understanding the Value of an LLM-Based Voting Advice Application Chatbot](http://arxiv.org/abs/2505.09806v1), 2025-05-14

(73) [Contextual Phenotyping of Pediatric Sepsis Cohort Using Large Language Models](http://arxiv.org/abs/2505.09805v1), 2025-05-14

(74) [A Multimodal Multi-Agent Framework for Radiology Report Generation](http://arxiv.org/abs/2505.09787v1), 2025-05-14

(75) [A Survey on Large Language Models in Multimodal Recommender Systems](http://arxiv.org/abs/2505.09777v1), 2025-05-14

(76) [Trustless Autonomy: Understanding Motivations, Benefits and Governance Dilemma in Self-Sovereign Decentralized AI Agents](http://arxiv.org/abs/2505.09757v1), 2025-05-14

(77) [FAS-LLM: Large Language Model-Based Channel Prediction for OTFS-Enabled Satellite-FAS Links](http://arxiv.org/abs/2505.09751v1), 2025-05-14

(78) [Achieving Tokenizer Flexibility in Language Models through Heuristic Adaptation and Supertoken Learning](http://arxiv.org/abs/2505.09738v1), 2025-05-14

(79) [An AI-Powered Research Assistant in the Lab: A Practical Guide for Text Analysis Through Iterative Collaboration with LLMs](http://arxiv.org/abs/2505.09724v1), 2025-05-14

(80) [VeriFact: Enhancing Long-Form Factuality Evaluation with Refined Fact Extraction and Reference Facts](http://arxiv.org/abs/2505.09701v1), 2025-05-14

(81) [Customizing a Large Language Model for VHDL Design of High-Performance Microprocessors](http://arxiv.org/abs/2505.09610v1), 2025-05-14

(82) [Adversarial Suffix Filtering: a Defense Pipeline for LLMs](http://arxiv.org/abs/2505.09602v1), 2025-05-14

(83) [How Hungry is AI? Benchmarking Energy, Water, and Carbon Footprint of LLM Inference](http://arxiv.org/abs/2505.09598v1), 2025-05-14

(84) [WorldView-Bench: A Benchmark for Evaluating Global Cultural Perspectives in Large Language Models](http://arxiv.org/abs/2505.09595v1), 2025-05-14

(85) [Beyond Likes: How Normative Feedback Complements Engagement Signals on Social Media](http://arxiv.org/abs/2505.09583v2), 2025-05-14

(86) [Ethics and Persuasion in Reinforcement Learning from Human Feedback: A Procedural Rhetorical Approach](http://arxiv.org/abs/2505.09576v1), 2025-05-14

(87) [MIGRATION-BENCH: Repository-Level Code Migration Benchmark from Java 8](http://arxiv.org/abs/2505.09569v1), 2025-05-14

(88) [System Prompt Optimization with Meta-Learning](http://arxiv.org/abs/2505.09666v1), 2025-05-14

(89) [Tales of the 2025 Los Angeles Fire: Hotwash for Public Health Concerns in Reddit via LLM-Enhanced Topic Modeling](http://arxiv.org/abs/2505.09665v1), 2025-05-14

(90) [PT-MoE: An Efficient Finetuning Framework for Integrating Mixture-of-Experts into Prompt Tuning](http://arxiv.org/abs/2505.09519v1), 2025-05-14

(91) [Layered Unlearning for Adversarial Relearning](http://arxiv.org/abs/2505.09500v1), 2025-05-14

(92) [Card Sorting Simulator: Augmenting Design of Logical Information Architectures with Large Language Models](http://arxiv.org/abs/2505.09478v1), 2025-05-14

(93) [Deploying Foundation Model-Enabled Air and Ground Robots in the Field: Challenges and Opportunities](http://arxiv.org/abs/2505.09477v1), 2025-05-14

(94) [Analog Foundation Models](http://arxiv.org/abs/2505.09663v1), 2025-05-14

(95) [Omni-R1: Do You Really Need Audio to Fine-Tune Your Audio LLM?](http://arxiv.org/abs/2505.09439v1), 2025-05-14

(96) [Evaluating GPT- and Reasoning-based Large Language Models on Physics Olympiad Problems: Surpassing Human Performance and Implications for Educational Assessment](http://arxiv.org/abs/2505.09438v1), 2025-05-14

(97) [CXMArena: Unified Dataset to benchmark performance in realistic CXM Scenarios](http://arxiv.org/abs/2505.09436v1), 2025-05-14

(98) [Endo-CLIP: Progressive Self-Supervised Pre-training on Raw Colonoscopy Records](http://arxiv.org/abs/2505.09435v1), 2025-05-14

(99) [Large Language Models Are More Persuasive Than Incentivized Human Persuaders](http://arxiv.org/abs/2505.09662v1), 2025-05-14

(100) [SafePath: Conformal Prediction for Safe LLM-Based Autonomous Navigation](http://arxiv.org/abs/2505.09427v2), 2025-05-14
